src/components/Editable.tsx

```diff
@@ -451,7 +452,9 @@ const Editable = ({
 
     // NOTE: When Subthought components are re-rendered on edit, change is called with identical old and new values (?) causing an infinite loop
     const oldValue = oldValueRef.current
-    const newValue = e.target ? addEmojiSpace(unescape(strip(e.target.value, { preserveFormatting: true }))) : oldValue
+    const newValue = e.target
+      ? stripEmptyFormattingTags(addEmojiSpace(unescape(strip(e.target.value, { preserveFormatting: true }))))
+      : stripEmptyFormattingTags(oldValue)
```

Comment:
`stripEmptyFormattingTags` does not need to be called on `oldValue`, since `oldValue` merely contains the last `newValue` and is already stripped.